updates/src/process.js

```diff
@@ -20,30 +21,95 @@ function removedForVersion(version) {
 function mapStandaloneCompat(engines, { support, ...compat }) {
   return {
     ...compat,
-    engines: [
-      ...new Set(
-        Object.entries(support || {})
-          .map(([browser, s]) => {
-            const { version, engine } = engines[browser] || {};
-            const removed = s.find(({ version_removed }) =>
-              Boolean(version_removed)
-            );
-            const added = s.find(({ version_added }) => Boolean(version_added));
-            if (
-              added &&
-              gte(version, added.version_added) &&
-              !(removed && gte(version, removed.version_removed))
-            ) {
-              return engine;
-            }
-            return null;
-          })
-          .filter(Boolean)
-      ),
-    ],
+    engines: mapEngines(engines, { support }),
```

Comment:
Is there a reason we maintain the old "at current time" engines logic for standalone, rather than switching to the new "at that browser's release date" logic?